actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1254,7 +1254,8 @@ def text_area(object_name, method, options = {})
       # Returns a checkbox tag tailored for accessing a specified attribute (identified by +method+) on an object
       # assigned to the template (identified by +object+). This object must be an instance object (@object) and not a local object.
       # It's intended that +method+ returns an integer and if that integer is above zero, then the checkbox is checked.
-      # Additional options on the input tag can be passed as a hash with +options+. The +checked_value+ defaults to 1
+      # Additional options on the input tag can be passed as a hash in the +options+ parameter: To force the checkbox to be checked
+      # pass +checked: true+ in the options hash. You may pass HTML options there as well. The +checked_value+ defaults to 1
```

Comment:
How about a small tweak for conciseness?

```suggestion
      # Additional options on the input tag, including HTML options, can be passed as a hash with +options+. To force 
      # the checkbox to be checked, pass <code>checked: true</code> in the +options+ hash. The +checked_value+ defaults to 1
```